tests/Io/SapiHandlerTest.php

```diff
@@ -265,16 +267,37 @@ public function testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromSt
             $this->markTestSkipped('Test requires running phpunit with --stderr and Xdebug enabled');
         }
 
+        header_remove();
+        $_SERVER['SERVER_PROTOCOL'] = 'http/1.1';
+        $sapi = new SapiHandler();
+        $body = new ThroughStream();
+        $response = new Response(200, [], $body);
+
+        $this->expectOutputString('test');
+        $sapi->sendResponse($response);
+
+        $this->assertEquals(['Content-Type:'], xdebug_get_headers());
+
+        $body->end('test');
+    }
+
+    public function testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromStreamDataAndNoBufferHeaderForNginxServer(): void
+    {
+        if (headers_sent() || !function_exists('xdebug_get_headers')) {
+            $this->markTestSkipped('Test requires running phpunit with --stderr and Xdebug enabled');
+        }
+
+        header_remove();
         $_SERVER['SERVER_PROTOCOL'] = 'http/1.1';
+        $_SERVER['SERVER_SOFTWARE'] = 'nginx/1';
         $sapi = new SapiHandler();
         $body = new ThroughStream();
         $response = new Response(200, [], $body);
 
         $this->expectOutputString('test');
         $sapi->sendResponse($response);
 
-        $previous = ['Content-Type:', 'Content-Length: 2'];
-        $this->assertEquals(array_merge($previous, ['Content-Type:']), xdebug_get_headers());
+        $this->assertEquals(['Content-Type:', 'X-Accel-Buffering: no'], xdebug_get_headers());
 
         $body->end('test');
     }
```

Comment:
I can understand that you moved the `testSendResponseSendsStreamingResponseWithNoHeadersAndBodyFromStreamDataAndNoBufferHeaderForNginxServer` test, but it messes up the diff in GitHub and I'd say it's not necessary for this PR. I would suggest to
leave this test at its current position, WDYT?